shared/typeinference/codeql/typeinference/internal/TypeInference.qll

```diff
@@ -635,28 +646,36 @@ module Make1<LocationSig Location, InputSig1<Location> Input1> {
         }
       }
 
+      /**
+       * Holds if the type of `a` at `apos` has the base type `base`, and when
+       * viewed as an element of that type has at `path` the type `t`.
```

Comment:
has at `path` the type `t` -> has the type `t` at `path`.

Reply:
Yeah, let's change it. I was trying to phrase this to match the order of the parameters, but it is clumpsy.